### ISSUE TITLE
G'MIC

### ISSUE THREAD

**Issue by olear** [olear/openfx-arena#8]
Never tried G'MIC before, but looks ok. Test and add some plugins.

**Comment by olear** [comment]
and/or others.

**Comment by olear** [comment]
got an poc plugin, just need to read/write the pixels right.

**Comment by olear** [comment]
The  CeCILL license confuses me.

If I have read wikipedia and their homepage, I can only include software with  CeCILL-B or  CeCILL-C. This excludes GMIC since it's regular  CeCILL.

**Comment by olear** [comment]
CImg.h is CeCILL-C (but the rest is not), that's more like LGPL, and I don't want that. So basicly GMIC/CImg are LGPL and GPL.....ish

Don't see the point in CeCILL, just adds more confusion.

**Comment by devernay** [comment]
Ole, David (GMIC main author and maintainer) is not opposed to having gmic in an openfx plugin, although theoretically loading a GPL (or Cecill) plugin is only possible in GPL software.
What we did is a separate https://github.com/MrKepzie/openfx-gmic project, which will be GPL or Cecill, and distribute it separately from Natron and other OpenFX plugins. David thought it was OK.

We (@MrKepzie and I, you are welcome to contribute) are planning to do:
- a plugin that accepts gmic expressions/scripts, possibly with several inputs (as we did with SeExpr)
- a set of plugin generated automatically from the GIMP plugin configuration file, or the gmicol XML file https://gmicol.greyc.fr/
